rpm/update-spec-version.sh

```diff
@@ -8,7 +8,7 @@ set -eox pipefail
 
 PACKAGE=podman
 SPEC_FILE=$PACKAGE.spec
-GIT_DESCRIBE=$(git describe)
+GIT_DESCRIBE=$(grep '^const RawVersion' ../version/rawversion/version.go | cut -d\" -f2)
```

Comment:
maybe rename the variable as it no longer uses git describe for anything?

Reply:
done.